### PR TITLE
config: add config option for bloom filter

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT
Signed-off-by: Jakub Sztandera <kubuxu@protonmail.ch>

### CODE REVIEW

**Comment by whyrusleeping** [comment]
LGTM, thanks

File: repo/config/init.go [hunk 1]
```diff
@@ -85,6 +85,7 @@ func datastoreConfig() (Datastore, error) {
 		StorageGCWatermark: 90, // 90%
 		GCPeriod:           "1h",
 		HashOnRead:         false,
+		BloomFilterSize:    0,
```
Comment (jbenet) [inline repo/config/init.go]: so this is disabled by default?

Comment (Kubuxu) [inline repo/config/init.go]: Yes it is, we decided to do that until we get better metrics, repo size caching (important information to size to bloom filter correctly) and bloom filter expansion.

Comment (jbenet) [inline repo/config/init.go]: SGTM

**Pull request merged**